package.json

```
@@ -48,6 +48,6 @@
     "http2-wrapper": "~2.2"
   },
   "peerDependencies": {
-    "axios": "~1.7"
+    "axios": "~1.8"
```

axios v1.9 is already out, maybe this line could be changed as this, to prevent updating this line every time.
```suggestion
    "axios": "^1.9"
```